### DISCUSSION TITLE
ChunkLoadError: Loading chunk xxxx(number) failed.

### DISCUSSION THREAD

**Discussion by nongtiny** [vercel/next.js#68970] [Help]
### Summary

It found this error in my Monitoring using Datadog, and I can't reproduce this in the local.

This error appear after I upgraded nextjs to version `14.2.5`

<img width="1012" alt="Screenshot 2567-08-16 at 15 24 44" src="https://github.com/user-attachments/assets/c2276ccd-26fe-4f56-8a42-c0822762de98">

### Additional information

```js
Device Type: Mobile
OS Name: Android
OS Version: 10

Browser
name: Chrome Mobile
version: 127.0.0.0
version_major: 127

Next info:


Binaries:
  Node: 20.16.0
  npm: 10.8.1
  Yarn: 1.22.22
  pnpm: 9.6.0
Relevant Packages:
  next: 14.2.5 // Latest available version is detected (14.2.5).
  eslint-config-next: 11.1.2
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.5.4
Next.js Config:
  output: N/A
```

**Comment by dbstjddbwls** [comment]
I'm also having this issue. Is there a problem with `v14.2.5`?

**Reply by nongtiny** [reply]
Thanks for the reply brother.

This error still happen in my website. Did you still get this error?

**Comment by EvgeniyPanin** [comment]
Also encounter this bag in "next": "14.2.15"

Receiving an error in sentry:
Loading chunk 6654 failed.
(missing: https://allo.tochka.com/_next/static/chunks/6654-0f457d22eaaba3d2.js)
despite the fact, that the chunk does exist in every kubernetes pod and i see in kibana logs that every request for this chunk has been successfully fulfilled with 200 or 304 status.

**Reply by abhibhaw** [reply]
Did u find a solution @EvgeniyPanin ?
